stop_and_wait_plugin/config/parameters.yaml

```diff
@@ -26,7 +26,7 @@ centerline_sampling_spacing: 1.0
 # Double: The default buffer in meters used for where the vehicle can come to a stop during execution. 
 #         This value is overriden by the first value in maneuver.parameters.float_valued_meta_data[]
 # Units: m
-default_stopping_buffer: 5.0
+default_stopping_buffer: 15.0
```

Comment:
Same question on acceptability as above.

Reply:
This parameter is being overridden, so we can actually keep the older value. So I reversed it.